### PR TITLE
react-native-sdk: Add attributes to WherebyEmbed component

### PR DESCRIPTION
**Pull request by thyal**
### Description
Adds attributes/preferences to the embed element. For now it's just copied over from the browser-sdk embed element, but this should be moved to the core package so that it can be reused across both libraries.

**Summary:**

**Related Issue:**

### Checklist

-   [x] My code follows the project's coding standards.
-   [x] Prefixed the PR title and commit messages with the service or package name
-   [ ] I have written unit tests (if applicable).
-   [ ] I have updated the documentation (if applicable).
-   [x] By submitting this pull request, I confirm that my contribution is made
        under the terms of the MIT license.

### CODE REVIEW

File: apps/react-native-webview-example/app/room/index.tsx [hunk 1]
```diff
@@ -1,8 +1,7 @@
 import * as React from "react";
 import { ScrollView, StyleSheet, Text, View } from "react-native";
 import { WherebyEmbed, WherebyEmbedRef } from "@whereby.com/react-native-sdk/embed";
-const ROOM_URL = "";
-const queryParams = "skipMediaPermissionPrompt";
+const ROOM_URL = "https://team.whereby.com/havardholvik";
```
Comment (remi-quinto) [inline apps/react-native-webview-example/app/room/index.tsx:4]: Is this room exposed intentionally ?

File: apps/react-native-webview-example/app/room/index.tsx [hunk 2]
```diff
@@ -21,12 +20,15 @@ export default function Room() {
                     <Text key={index}>{entry}</Text>
                 ))}
             </ScrollView>
+            {/* <Button onPress={() => wherebyRoomRef.current?.knock()} title="Knock" /> */}
```
Comment (remi-quinto) [inline apps/react-native-webview-example/app/room/index.tsx:23]: Looks like leftover from other ticket.

File: apps/react-native-webview-example/app/room/index.tsx [hunk 3]
```diff
@@ -21,12 +20,15 @@ export default function Room() {
...
             <View style={{ flex: 1, height: "100%" }}>
                 <WherebyEmbed
                     ref={wherebyRoomRef}
                     style={styles.container}
-                    roomUrl={ROOM_URL}
-                    queryParams={queryParams}
+                    room={ROOM_URL}
+                    minimal={"on"}
```
Comment (remi-quinto) [inline apps/react-native-webview-example/app/room/index.tsx:29]: Maybe add a comment why the minimal configuration is suggested on mobile, or just as en example of how to use properties.

File: apps/react-native-webview-example/app/room/index.tsx [hunk 4]
```diff
@@ -21,12 +20,15 @@ export default function Room() {
...
+                    skipMediaPermissionPrompt={"on"}
+                    displayName="React Native Client"
```
Comment (remi-quinto) [inline apps/react-native-webview-example/app/room/index.tsx:31]: Is this a placeholder display name ? Or something for us to track?

File: packages/react-native-sdk/src/embed/index.tsx [hunk 5]
```diff
@@ -1,25 +1,162 @@
...
+            if (!roomUrl.searchParams.has(k)) {
+                roomUrl.searchParams.set(k, v);
+            }
+        });
 
         return (
             <WebView
                 ref={ref}
-                source={{ uri: url }}
+                source={{ uri: roomUrl.href }}
```
Comment (remi-quinto) [inline packages/react-native-sdk/src/embed/index.tsx:155]: 👍

**Review by remi-quinto — CHANGES_REQUESTED** [review]
I've tested some properties like "minimal" on/off and it works as expected. 
Let's not expose havard room URL and maybe add some comments on the properties example in the demo app and this is good to go.

File: apps/react-native-webview-example/app/room/index.tsx (hunk 1 above)
Comment (thyal) [reply apps/react-native-webview-example/app/room/index.tsx]: Nope! Removed now.

File: apps/react-native-webview-example/app/room/index.tsx (hunk 2 above)
Comment (thyal) [reply apps/react-native-webview-example/app/room/index.tsx]: Removed

File: apps/react-native-webview-example/app/room/index.tsx (hunk 4 above)
Comment (thyal) [reply apps/react-native-webview-example/app/room/index.tsx]: Just a placeholder. Removed it for now. We can add more example props in this app later if needed

**Review by remi-quinto — APPROVED** [review]
LGTM !

**Pull request merged**